projects/README.md

```diff
@@ -2,14 +2,14 @@
 
 |project|license|description|version|repository|patch/diff|
 |-------|-------|-----------|-------|----------|----------|
+|[cppzmq](https://github.com/zeromq/cppzmq 'cppzmq')|[MIT](https://github.com/zeromq/cppzmq/blob/v4.2.1/LICENSE 'MIT')|cppzmq is a minimal c++ binding to the libzmq functions.|4.2.1|[repo](https://github.com/zeromq/cppzmq/tree/v4.2.1 'cppzmq repo on github')|none|
 |[GLEW](http://glew.sourceforge.net/ 'The OpenGL Extension Wrangler Library')|[open](http://glew.sourceforge.net/credits.html 'Modified BSD, Mesa 3-D (MIT), and Khronos (MIT)')|The OpenGL Extension Wrangler Library (GLEW) is a cross-platform open-source C/C++ extension loading library.|1.13.0|[repo](https://github.com/nigels-com/glew 'GLEW repo on github')|none|
 |[HDF-GROUP](http://www.hdfgroup.org 'The HDF Group')|[open](https://www.hdfgroup.org/ftp/HDF5/current/src/unpacked/COPYING 'HDF5 License')|HDF5 is a data model, library, and file format for storing and managing data|1.8.16|none|[patch](../patches/hdf5.patch 'patch')|
-|[Kerberos](http://web.mit.edu/kerberos/ 'Kerberos')|[open](http://web.mit.edu/kerberos/krb5-1.14/doc/mitK5license.html 'MIT Kerberos License information')|Kerberos is a network authentication protocol (only built on Windows)|5.1.14|none|none|
-|[librdkafka](https://github.com/edenhill/librdkafka 'librdkafka on github')|[open](https://github.com/edenhill/librdkafka/blob/master/LICENSE '2-clause BSD license')|librdkafka is a C library implementation of the Apache Kafka protocol, containing both Producer and Consumer support|0.9.5|[repo](https://github.com/distributePro/librdkafka 'distributePro fork of librdkafka repo on github')|[diff](https://github.com/distributePro/librdkafka/compare/edenhill:v0.9.5...xp-v0.9.5 'patch/diff') [win-diff](https://github.com/distributePro/ovsrpro/blob/master/patches/librdkafka-windows.patch 'patch/diff (Windows only)')|
+|[Kerberos](http://web.mit.edu/kerberos/ 'Kerberos')|[open](http://web.mit.edu/kerberos/krb5-1.14/doc/mitK5license.html 'MIT Kerberos License information')|Kerberos is a network authentication protocol (only built on Windows)|4.0.1|none|none|
```

Comment:
@swflb looks like you deleted your comment regarding the Kerberos versions, but I'll attempt an answer anyway (I'm sure you figured it out enough to satisfy yourself, but I figured I'd document it here in case I have to answer it again and I won't remember)

the version that shows up in the README.md is 4.0.1 since that's the version built on Windows and we only build this project on Windows and the cmake-generated README doesn't have a way to specify two versions from the same project file (I did add a note to the `kerberos.cmake` file that gives an example of having two versions -- in two different project files -- as we currently do in externpro for ffmpeg) -- and I didn't spend much time looking for a 4.0.1 version of the license, so it is hard-coded currently to the krb5-1.14 version...